en/physics/physics-rigidbody.md

```diff
@@ -1,6 +1,6 @@
 # RigidBody Component
 
-**RigidBody** is the basic object that make up the physical world, and it can make a node physically affected and react. The RigidBody component is invalid when using the **Builtin** physics engine.
+**RigidBody** is the basic object that make up a physics` world, and it can make a node physically affected and react. The RigidBody component is invalid when using the **Builtin** physics engine.
```

Comment:
there is an extraneous backtick.

**RigidBody** is the basic object that make up a physics` world, and it can make a node physically affected and react. The RigidBody component is invalid when using the **Builtin** physics engine.
-->
**RigidBody** is the basic object that makes up a physics world. It can make a node physically affected and react accordingly. The **RigidBody** component is invalid when using the **Builtin** physics engine.